src/routes/files/handlers.js

```diff
@@ -0,0 +1,103 @@
+import { v4 as uuidv4 } from 'uuid';
+import handleErrors from '../../lib/apiErrorHandler';
+import { sequelize, File } from '../../models';
+import s3Uploader from '../../lib/s3Uploader';
+
+const fs = require('fs');
```

Comment:
Can you change these to ES6 `import` for consistency?

Reply:
Yup